### ISSUE TITLE
[feat]: Zoom without click on the image

### ISSUE THREAD

**Issue by manel00** [SimonGolms/ngx-ionic-image-viewer#37]
Hello,

Is it possible to ask for to do zoom on the image without click on it?

Thank you

**Comment by SimonGolms** [comment]
Hi @manel00, can you please explain this to in a more detail way, or provide an example? At the moment I don't understand what you mean here.

**Comment by manel00** [comment]
Yes i meant, i need to do click on the image and then its fullscreen with the image and can play with the zoom, but not able
